storage/Utils/SystemCmd.cc

```diff
@@ -248,26 +272,35 @@ namespace storage
 	}
 	if ( !_testmode && ok )
 	{
-	    _pfds[0].fd = sout[0];
+	    _pfds[0].fd = sin[1];
```

Comment:
I would prefer PIPE_READ for 0 and PIPE_WRITE for 1, but surprisingly they are not defined in unistd.h :(
(so this would become `_pfds[STDIN_FILENO].fd = sin[PIPE_WRITE];`)